_dashboards/discover/multi-data-sources.md

```diff
@@ -41,22 +41,45 @@ To create a new data source connection:
+    
+    When you select the authentication method, the applicable fields appear for the selected method. Enter the required details.
+
+    After all of the required fields have the appropriate details entered, the **Test connection** and **Create data source connection** buttons become active. You can select **Test connection** to confirm the connection is valid.
+
+1. Select **Create data source connection** to save your settings. The connection is created. The active window returns to the **Data Sources** main page, and the new connection appears in the list of data sources.
 
     <img src="{{site.url}}{{site.baseurl}}/images/multi-data-sources-3.png" width=500 alt="Data sources list user interface">
 
-1. Optional: Select the data source to verify that the settings are configured properly, as shown in the following image.
+    You can also delete the data source connection from this page by selecting the checkbox to the left of the title and then selecting **Delete 1 connection** to the right of the search bar. Selecting multiple checkboxes for multiple connections is also supported.
+    {: .note }
+
+### Editing and updating a data source connection
+
+To make changes to the data source connection, select the connection in the list on the **Data Sources** main page. The connection details window opens, as shown in the following image.
+
+<img src="{{site.url}}{{site.baseurl}}/images/multi-data-sources-3.5.png" width=500 alt="Data sources settings verification and editing user interface">
+
+To make changes to **Connection Details**, edit one or both of the **Title** and **Description** fields and select **Save changes** in the lower right corner of the screen. You can also cancel changes here. To change the **Authentication Method**, select a different authentication method and enter your credentials if applicable, then select **Save changes** in the lower right corner of the screen. The changes are saved.
+
+When **Username & Password** is the selected authentication method, you can update the password by selecting **Update stored password** beside the **Password** field. In the popup window, enter a a new password in the first field and then enter it again in the second field to confirm. Select **Update stored password** in the popup window. The new password is saved. Select **Test connection** in the upper right corner of the screen to confirm the connection is valid.
```

Comment:
Does the UI actually use the ampersand? We generally use "and" spelling out. If that's the way it's displayed, then disregard my comment.

Reply:
Yeah, in the first photo. But I agree with your point.